src/alasqlparser.jison

```diff
@@ -589,6 +609,21 @@ ArrowDot
 	| DOT
 	;
 
+IntervalLiteral
+	: MICROSECOND
+	| MILLISECOND
+	| SECOND
+	| MINUTE
+	| HOUR
+	| DAY
+	| DAYOFWEEK

```

Comment:
I dont think this is a valid interval. Can you please confirm?

Reply:
Weekday can be used as interval, not dayofweek